routes/items.js

```diff
@@ -17,12 +17,25 @@ router.get( '/', ( request, response ) => {
     .then( respondWithItems( user, data => response.render( 'items/index', data )))
 })
 
+
+router.get( '/download/:type', ( request, response ) => {
+  const { Item } = request.app.get( 'models' )
+
+  const { user, query } = request
+
+  response.set('Content-Type', 'text/plain')
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```